### DISCUSSION TITLE
Can I display none database-plugin__embed-link-wrapper class?

### DISCUSSION THREAD

**Discussion by JoshKasap** [RafaelGB/obsidian-db-folder#645] [Q&A]
### Contact Details

JoshKasap@gmail.com

### What happened?

In my custom theme, due to the way I style links, this plugin seems to make a blank link to the current page. the link is contained within a div with the class `database-plugin__embed-link-wrapper` 

![image](https://user-images.githubusercontent.com/56367532/205380199-57135086-60cd-404f-b0a5-d2146aa976b5.png)

The HTML code for this appears in the default theme as well but it has a pixel height and width of 0px so it doesn't actually appear or mess with layout. 

If I give this class a style of `display: none` then it seems to fix this issue. alternatively I could give it a width and height of 0px to get the same result

![image](https://user-images.githubusercontent.com/56367532/205380984-d4f2e12d-594c-4eed-9b14-2354b455b10a.png)

So I have three questions
1. Would adding a display none to this class somehow cause issues for the plugin's functionality? 
2. Would setting height and width to 0px be a better option to avoid potential issues? 
3. What does this class/html code even do? 

### What platform were you using?

Desktop

### Version of the plugin

3.0.0

### Are you using the latest version of the plugin in your Obsidian vault?

- [X] I have verified that I am on the latest version

### Are you check if there is a similar issue?

- [X] I have verified that there are not similar issues

**Comment by RafaelGB** [comment]
Hi @JoshKasap ,
that css it's some legacy code to enable the click events on empty cells. Remove it could potentially make the empty cells unclickeable. I am sire there is a better way to fix this problem without affects the css

**Comment by RafaelGB** [comment]
moved to discussion
